### PR TITLE
exit if there are no jobs to rehearse

### PR DESCRIPTION
**Pull request by droslean**

### CODE REVIEW

File: cmd/pj-rehearse/main.go [hunk 1]
```diff
@@ -148,6 +148,10 @@ func main() {
 	}
 
 	changedPresubmits := diffs.GetChangedPresubmits(prowConfig, prowPRConfig, logger)
+	if len(changedPresubmits) == 0 {
+		logger.Info("no jobs to rehearse have been found")
+		gracefulExit(o.noFail, "")
```
Comment (petr-muller) [inline cmd/pj-rehearse/main.go]: This would error-out on no rehearsed jobs, we probably do not want that.

Comment (droslean) [reply cmd/pj-rehearse/main.go]: addressed

**Review by petr-muller — APPROVED** [review]
/lgtm

**Pull request merged**